client/export_test.go

```diff
@@ -49,3 +49,5 @@ func UnmarshalSnapshotAction(body io.Reader) (act snapshotAction, err error) {
 	err = json.NewDecoder(body).Decode(&act)
 	return
 }
+
+type DownloadData downloadData
```

Comment:
you want a *type alias*, not a new type. I.e.,
```suggestion
type DownloadData = downloadData
```